cluster/addons/dns/coredns.yaml.sed

```diff
@@ -95,7 +95,7 @@ spec:
           operator: "Exists"
       containers:
       - name: coredns
-        image: coredns/coredns:1.0.1
+        image: k8s.gcr.io/coredns:1.0.1
```

Comment:
This seems like an older coreDNS version.

Reply:
Yes, this is the version for k8s release 1.9